test/specs/rules/require-numeric-literals.js

```diff
@@ -0,0 +1,37 @@
+var Checker = require('../../../lib/checker');
+var assert = require('assert');
+
+describe('rules/require-numeric-literals', function() {
+    var checker;
+
+    beforeEach(function() {
+        checker = new Checker();
+        checker.registerDefaultRules();
+
+        checker.configure({
+            esnext: true,
+            requireNumericLiterals: true
+        });
+    });
+
+    it('should report on use of parseInt', function() {
+        assert(checker.checkString('parseInt("111110111", 2) === 503;').getErrorCount() === 1);
+        assert(checker.checkString('parseInt("767", 8) === 503;').getErrorCount() === 1);
+        assert(checker.checkString('parseInt("1F7", 16) === 255;').getErrorCount() === 1);
+    });
+
+    it('should not report on numeric literals', function() {
+        assert(checker.checkString('0b111110111 === 503;').isEmpty());
+        assert(checker.checkString('0o767 === 503;').isEmpty());
+        assert(checker.checkString('0x1F7 === 503;').isEmpty());
+    });
+
+    it('should not report on other call expressions', function() {
+        assert(checker.checkString('a();').isEmpty());
+        assert(checker.checkString('a(1);').isEmpty());
+        assert(checker.checkString('a(1, 2);').isEmpty());
+        assert(checker.checkString('parseInt(1, 2);').isEmpty());
```

Comment:
Without proper use cases from the community, it feels like we're guessing as to what people need. 

Are you saying that this rule isn't worth including?